app/controllers/search/results_controller.rb

```diff
@@ -1,9 +1,13 @@
 class Search::ResultsController < ApplicationController
 
   def index
-    @obituaries_api = ObituarySearch.new.advanced_query(params) || []
+    if params[:name].empty? && params[:location].empty? && params[:age].empty? && params[:year].empty?
```

Comment:
If no entries are given redirects back to advanced search page

Reply:
Hope this fixes it! Thanks for working on it Eric.